test/bioclip/test_queries.py

```
@@ -4,13 +4,13 @@
 import numpy as np
 import pandas as pd
 
-from camvidlog2.data import (
+from camvidlog2.bioclip.data import (
     EmbeddingCollection,
     EmbeddingGroup,
     FrameEmbedding,
     StringEmbedding,
 )
```

Pylint (3.3.7)

[error] Unable to import 'camvidlog2.bioclip.data'

(E0401)